client/src/Plant/PlantList.js

```diff
@@ -0,0 +1,39 @@
+import {useEffect, useState} from 'react';
+import {Link} from 'react-router-dom';
+import Api from '../Api';
+
+function PlantList(){
+    const [plants, setPlants] = useState([]);
+    useEffect(function(){
+      Api.plants.index().then(response => setPlants(response.data));  
+    }, []);
+
+    function onDelete(section){
+      if (window.confirm(`Are you sure you wish to delete ${plant.name}?`)){
+        //we'll execute code to delete the section
+        Api.plants.delete(plant.id).then(function(){
+          // we're filtering the sections list, keeping every section that does not 
+          //match the one we're deleting
+          const newPlants = plants.filter(s => s.id !== plant.id);
+          setPlants(newPlants);
+        });
+      }
+    }
+    return(
+      //display
+        <main className="continer">
+            <h1>Plants List</h1>
+            <Link className="btn btn-primary" to="/plants/new">New</Link>
+            <ul>
+              {plants.map(s =>(
+                  <li>
+                    <p><Link to={`/plants/${s.id}/edit`}>{s.name},{s.slug},{s.position}</Link></p>
```

Comment:
Again, slug and position are not part of the Plant model. While it is a perfectly fine approach to start by copying code from other models, you do need to stop and make changes for what is different in this model.